tensorflow/core/grappler/optimizers/meta_optimizer.cc

```diff
@@ -369,12 +372,21 @@ Status MetaOptimizer::InitializeOptimizers(
         MakeUnique<AutoMixedPrecision>(AutoMixedPrecisionMode::CUDA));
   }
 #ifdef INTEL_MKL
+  if (AutoMixedPrecisionEnabled(cfg_.auto_mixed_precision_bfloat16()) &&
+      AutoMixedPrecisionEnabled(
+          plugin_configs.toggle_config["auto_mixed_precision_bfloat16"]) &&
+      IsMKLEnabled()) {
+    optimizers->push_back(
+        MakeUnique<AutoMixedPrecision>(AutoMixedPrecisionMode::BF16));
+  }
   if (AutoMixedPrecisionEnabled(cfg_.auto_mixed_precision_mkl()) &&
       AutoMixedPrecisionEnabled(
           plugin_configs.toggle_config["auto_mixed_precision_mkl"]) &&
       IsMKLEnabled()) {
+    LOG_FIRST_N(INFO, 1) << "NOTE: auto_mixed_precision_mkl is deprecated. "\
```

Comment:
Use WARNING instead of INFO. Also, no need for the backslash at the end of this line, nor do you need an explicit newline `\n` at the end.